### ISSUE TITLE
Best BNS Thread competition!

### ISSUE THREAD

**Issue by Hero-Gamer** [stacksgov/decentralized-grants#35]

**Comment by Snazzy-888** [comment]
**1) Total grant amount:** $100

**2) Project Description:**
Twitter wide engagement initiative for BNS through a thread competition. This will help to educate, inform and inspire the Stacks & wider crypto ecosystem as to what BNS is.

For those taking part the aim will be for them to create the most interesting thread about BNS . This could be educational, involve the history of BNS, it could include metrics such as registrations, sales, dates of such data and so on.

The most interesting and captivating thread will be in with a chance of winning, with the winners being chosen by the community.

**3) Impact:**
This will result in more exposure and could increase the adoption of BNS.

Our target is for up to 10 community members/individuals within the Stacks ecosystem  to be involved in the thread competition.

**4) Project Timelines, Milestones, and Deliverables:**
Competition over 1 week, with the winner announced on the 8th day

Kick-off payment: 0% - $0

Milestone 1: 0% - $0 - Target completion date: 14th August 2024
Deliverable: 
-  The BNS Community & followers on X are prepared as to the commencement of the BNS Thread  on Wednesday 14th August (Competition the following week). Preparations via announcements and X posts.

- Milestone 2: 100% - $100 - Target completion date: 26th August 2024
Deliverable:
- The BNS Community will have been given 7 days to complete their BNS threads (Mon 19th - Sun 25th) and on the 8th day through discussion with the Core Team and BNS Community, 3 winners will be announced.

5**) Is / will this project be open-sourced?**
N/A

**6) ZeroAuthorityDAO profile link:**
https://zeroauthority.xyz/profiles/SPQNJ28CTBN4BX7SGZBKVAP8FRBXX759TEXQQCB2

**Comment by Snazzy-888** [comment]
BNS Thread Competition underway! 🎉

https://x.com/bns_community/status/1865803535943671832

**Comment by Hero-Gamer** [comment]
Hi Snazzy, 

From our DM conversation in discord, my understanding is that the entries (as of today) for the  prize giveaway has not been as many as expected, however there is a few days left to submit until giveaway deadline ends, so might still come in last minutes. 

I do wanna approve it, and will approve it. But just ask you to ensure no matter what you distribute the grant money ($100) this giveaway if all things go well next few days entries wise, or if not, in the future you would device another initiative to promote & market BNS using $100 in whatever way shape or form you feel will have the maximum impact. 

If you all agree. I will just approve this fund to close up this as it's quite small and not very risky and you are trust worthy in BNS community. 

This grant is approved. 
```
Milestone 2: 100% - $100 - Target completion date: 26th August 2024
Deliverable:
The BNS Community will have been given 7 days to complete their BNS threads (Mon 19th - Sun 25th) and on the 8th day through discussion with the Core Team and BNS Community, 3 winners will be announced.
```

Thanks for keeping the spirit of BNS community up Snazzy. 

@shaktistacks please could you help disburse the fund so Snazzy can re-distribute to the winners of the giveaway. Thanks a lot!

**Comment by Snazzy-888** [comment]
Yep of course, if this giveaway is unsuccessful I'll devise another $100 giveaway for the community! Maybe something shorter like asking the community to describe BNS in one sentence in the comments and share with their followers to enter... 🤔
